week-3/alarmclock/alarmclock.js

```diff
@@ -11,6 +61,8 @@ function setup() {
 
   document.getElementById("stop").addEventListener("click", () => {
     pauseAlarm();
+    clearInterval(bgIntervalId);
```

Comment:
You weren't really supposed to edit down here but the solution works so I'll give you that!

Reply:
I didn't want add an extra addListener and, used the existing one to stop flashing background :)